htdocs/compta/facture/class/api_invoices.class.php

```diff
@@ -477,16 +477,17 @@ public function postContact($id, $contactid, $type)
 	 *
 	 * @param int    $id             Id of invoice to update
 	 * @param int    $rowid          Row key of the contact in the array contact_ids.
+	 * @param string $type           Type of the contact (BILLING, SHIPPING, CUSTOMER).
 	 *
-	 * @url	DELETE {id}/contact/{rowid}
+	 * @url	DELETE {id}/contact/{rowid}/{type}
```

Comment:
Why adding this parameter $type ?
Delete should receive the rowid to delete to be able to delete ?

Reply:
If the same contact is on DELIVERY and INVOICE, it is he is unlinked from both positions. It is a real problem if we only have to change the INVOICE contact with a new contact without dissociating the DELIVERY contact. When I unlink a contact on the Dolibarr invoice contact card, I only unlink the billing or the delivery one, it seems logical to me that on the API, this behaves identically.

Reply:
@eldy i did not understand your point of view. it is the same reason as orders. https://github.com/Dolibarr/dolibarr/pull/14792#discussion_r493847596

Reply:
@eldy it's the same reasoning as with the orders API.